samples/client.c

```diff
@@ -868,13 +921,15 @@ migrate_from(int sock, size_t *nr_iters, struct iovec **migr_iters,
         .crcp = crcp
     };
 
+    size_t max_iter_size = 4096;
+
     ret = pthread_create(&thread, NULL, fake_guest, &fake_guest_data);
     if (ret != 0) {
         errno = ret;
         err(EXIT_FAILURE, "failed to create pthread");
     }
 
-    *nr_iters = 2;
+    *nr_iters = 8;
```

Comment:
Why?

Reply:
I'm quite confused about how the `nr_iters` stuff works - is it literally just a mechanism to ensure that the server cannot just keep sending data indefinitely to stop the client? In which case, would it make more sense to set `nr_iters` to something like `ceil(expected_data / max_iter_size)`?

Reply:
I _think_ `nr_iters` was used to avoid reading the entire migration data in one go, in order to make the sample a bit more realistic. This could have been better documented.
If you do have to change it, then please improve the (currently non-existent) documentation of `nr_iters` and also explain why it has to be changed.

Reply:
Okay, this makes sense.